python/ml4ir/applications/classification/model/classification_model.py

```diff
@@ -103,6 +93,56 @@ def evaluate(
             self.logger.info(f"Evaluation Results written at: {logs_dir}")
         return global_metrics, grouped_metrics, metrics_dict
 
+    @staticmethod
+    def get_chunks_from_df(dataframe, size):
+        """
+        Given a pd.DataFrame, creates batches of size `size`.
```

Comment:
Should we say we do this for performances reasons?

Reply:
yes. done.